Gopkg.toml

```diff
@@ -48,28 +53,32 @@ required = [
   version = "1.3.0"
 
 [[constraint]]
+  name = "sigs.k8s.io/controller-runtime"
+  version = "0.1.9"
+
+[[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.12.3"
+  version = "kubernetes-1.12.9"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.12.3"
-
-[[constraint]]
-  name = "k8s.io/client-go"
-  version = "kubernetes-1.12.3"
+  version = "kubernetes-1.12.9"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/code-generator"
-  version = "kubernetes-1.12.3"
+  version = "kubernetes-1.12.9"
 
-[[constraint]]
-  name = "sigs.k8s.io/controller-runtime"
-  version = "0.1.9"
+[[override]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.12.9"
+
+[[override]]
+  name = "k8s.io/apiextensions-apiserver"
+  version = "kubernetes-1.12.9"
 
 [[override]]
-  name="k8s.io/apiextensions-apiserver"
-  version="kubernetes-1.12.3"
+  name = "k8s.io/kubernetes"
+  version = "v1.13.3"
```

Comment:
Not sure if we can upgrade now. /cc @johnugeorge

Reply:
I tried, it works well.